docroot/modules/custom/va_gov_backend/va_gov_backend.install

```
@@ -183,3 +183,46 @@ function va_gov_backend_update_8003() {
     ]);
   }
 }
+
+/**
+ * Update the Owner for terms in health_care_service_taxonomy vocab.
+ */
+function va_gov_backend_update_8004() {
+  $updated = 0;
+  $skipped = 0;
+
+  // "National health care service descriptions" term ID.
+  $tid = 46;
+
+  $storage = \Drupal::entityTypeManager()->getStorage('taxonomy_term');
+  $terms = $storage->loadByProperties(['vid' => 'health_care_service_taxonomy']);
+
+  foreach ($terms as $term) {
+    // Check if owner is already set to term tid 46.
+    $owner = intval($term->field_owner->target_id);
+    if ($owner !== $tid) {
+      $term->set('field_owner', NULL);
+      $term->field_owner[] = ['target_id' => $tid];
+      try {
+        $saved = $term->save();
+        $updated = (is_int($saved) && $saved > 0) ? $updated + 1 : $updated;
+        Drupal::logger('va_gov_backend')->log(LogLevel::INFO, 'VHA health service taxonomy: Updated owner for term "%term_name".', [
+          '%term_name' => $term->label(),
+        ]);
+      }
+      catch (\Exception $e) {
+        $skipped = (is_int($saved) && $saved <= 0) ? $skipped + 1 : $skipped;
+        Drupal::logger('va_gov_backend')->log(LogLevel::ERROR, 'VHA health service taxonomy: Error updating owner for term "%term_name". Do it manually. Error message: %error_message.', [
+          '%term_name' => $term->label(),
+          '%error_message' => $e->getMessage(),
+        ]);
+      }
+    }
+  }
```

Shouldn't there be a $skipped incremented here too?  Otherwise we are only counting the ones that failed with an exception and not the terms that already had the right owner, which are the ones that are legitimately skipped.

---

The intent behind $skipped is to provide a number of actual failures for this specific message `Failed to update %skipped out of %count terms.`.  Perhaps I could have used $failed, instead of $skipped for clarity.

---

Ok, makes sense.  So our skipped are our failures and the true skipped are the delta between Updated + skipped, and count  :)  We have all the data we need.  :)